Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM maven:3.9.6-eclipse-temurin-21-alpine AS  build
+COPY ./ /src
+RUN mvn -f /src/pom.xml clean package
+
+FROM eclipse-temurin:21_35-jre-alpine
```

Comment:
Version 21_35 is an "old" version of Java version 21. Latest exact version number is 21.0.2+13 https://adoptium.net/temurin/archive/

So I suggest that we use 21.0.2_13-jre-alpine instead to have an exact tag which is prefered. If we want a tag that always tracks the latest 21 release we could instead use 21-jre-alpine.

Reply:
Updated version as suggested.